### ISSUE TITLE
4i

### ISSUE THREAD

**Issue by swetaC1711** [SuperSimpleDev/html-css-course-2022#65]
in in transition You have write four padding seconds ex- padding-right, padding-left, instead of that only padding 1s; is enough

**Comment by Titiksha-rgb** [comment]
that's right, but according to what you wrote, padding-1px would add padding in all four directions, that is, left,right,top,bottom, but he could have written as padding:0px 1px, where 0px is for top, bottom and 1px for left and right each.
